CONTRIBUTING.md

```diff
@@ -312,6 +312,7 @@ Stacked PRs (Y -> X -> `develop`) must never be merged until their base is final
 After merging the parent PR, child's base changes automatically (see [GitHub blogpost](https://github.blog/changelog/2020-05-19-pull-request-retargeting/))
 
 To create a release:
+* make sure the environment variables for the plugin signing are properly configured.
```

Comment:
I wouldn't put that particular step into the guide (in the same way we don't mention JetBrains Marketplace token in this guide). The releaser can assume that this is already configured in the CI. They don't need to have token and/or signing keys ready on their machine